Makefile

```diff
@@ -62,10 +62,10 @@ BITCODE_FILES = out/sqlite3.bc out/extension-functions.bc
 
 OUTPUT_WRAPPER_FILES = src/shell-pre.js src/shell-post.js
 
-OUTPUT_API_FILES = out/api.js
+OUTPUT_API_FILES = src/api.js
```

Comment:
I don't think we should put compiled results in `src/`

Reply:
Oh, ok, I get it. You put everything in `src/api.js` so that it doesn't have to be generated anymore. But you forgot to rename `OUTPUT_API_FILES`.

Reply:
ok, i'll rename it to SOURCE_API_FILES

Reply:
renamed to SOURCE_API_FILES in commit a32669a8bd1c5c555fd0aa8773824d9d7df4ab24